### PR TITLE
backend database migrations

### PR DESCRIPTION
**Pull request by mayur-rana**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,5 +5,15 @@
...
-  "license": "MIT"
+  "license": "MIT",
+  "scripts": {
+    "knex": "knex --knexfile server/db/knexfile.js"
+  },
+  "dependencies": {
+    "knex": "^0.14.6",
+    "sqlite3": "^4.0.0"
+  },
+  "devDependencies": {
```
Comment (don-smith) [inline package.json]: you can remove this section since it isn't being used.

File: server/db/migrations/20180503105052_creds.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (don-smith) [inline server/db/migrations/20180503105052_creds.js]: remove blank line

File: server/db/migrations/20180503105052_creds.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+exports.up = (knex, Promise) => {
+    return knex.schema.createTable('creds', table => {
+        table.increments('id').primary()
+        table.string('username')
+        table.string('hash')
+    })
+
```
Comment (don-smith) [inline server/db/migrations/20180503105052_creds.js]: remove blank line

File: server/db/migrations/20180503105145_users.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (don-smith) [inline server/db/migrations/20180503105145_users.js]: remove blank line

File: server/db/migrations/20180503105145_users.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+
+exports.up = (knex, Promise) => {
+    return knex.schema.createTable('users', (table) => {
+        table.increments('id').primary()
+        table.integer('cred_id').references('creds.id')
+        table.string('name')
+        table.text('order_text')
+    })
+
```
Comment (don-smith) [inline server/db/migrations/20180503105145_users.js]: remove blank line

**Comment by mayur-rana** [comment]
@don-smith  We've made these changes.

**Pull request merged**